### DISCUSSION TITLE
When should I expect VERDACCIO_STORAGE_PATH to be supported on an official docker?

### DISCUSSION THREAD

**Discussion by osher** [verdaccio/verdaccio#2992] [Q&A] [answered]
Hi, 

I'm trying to integrate verdaccio to a corporate gitlab-ci setup, where we want to use `verdaccio` for multi-stage build, and publish packages to the official corp registry only when all stages succeed.

We saw that the `verdaccio:latest` does not support `VERDACCIO_STORAGE_PATH`, however, we did get the expected behavior from `verdaccio:nightly-master`.

So the Q is:
What version does the support in `VERDACCIO_STORAGE_PATH` is planed to land in, and when can we see it as a part of an official build?

Thanks!

**Comment by juanpicado** [comment]
Depends of any want to contribute, here the PR for master, pretty straigforward for the branch 5.x https://github.com/verdaccio/verdaccio/pull/2199

**Reply by osher** [reply]
Excuse me, I'm missing something... I don't understand...
I see the PR has landed on master.

What's missing for it to become, for example docker image `verdaccio/verdaccio:5.5.3`?
Is it something a humble outside-contributor like me can do?

**Reply by juanpicado** [reply]
For context, master is not `5.x.x`, the version 5 is hosted on the branch `5.x`, master is meant to be the next major but not recommended used at this point since I usually release breaking changes with no fear. If you want that feature in the next v5 release has to be ported to the branch `5.x`. 

You can learn more here https://github.com/verdaccio/verdaccio/discussions/1690 and here https://github.com/verdaccio/verdaccio/discussions/2970

So what I'm asking you, do you want to contribute by porting what our friendly contributor made in master https://github.com/verdaccio/verdaccio/pull/2199 but in 5.x ? (caveat: the files, folders and might differ quite a lot since is completely different)

**Reply by osher** [reply]
I see. generally I don't touch typescript even with a 3m pole. but I'll have a look...

**Reply by juanpicado** [reply]
Np, well I actually did not notice there were bunch of new files, anyway, I have no much time these days but seems something cool to have on 5.x with no much pain.

**Reply by osher** [reply]
ok. I do think it should be streight forward, but do help me with this.

The master PR had 10 files, here's what I make of it:

master:  `.changeset/healthy-poets-compare.md`
5.x:  **_healthy poets?_** really 😄 😄 😄  I guess irrelevant in 5.x ❓ 

master: `docs/env.variables.md`
5.x:    `docs/env.variables.md` 👍 

master: `docs/env.variables.md`
5.x:    `docs/env.variables.md` 👍 

master: `packages/config/src/config.ts`
5.x:    `src/lib/config.ts` 👍  so far Streight forward 

but here it complicates:

master: 
```
 packages/config/test/config-parsing.spec.ts
 packages/config/test/config.spec.ts
 packages/config/test/package-access.spec.ts
 packages/config/test/partials/config/yaml/storage.yaml |
```
5.x:
 - `test/unit/modules/config/config.spec.ts` ...maybe ❓ ❓ 
 - LMK if there is more here...

master: `packages/config/test/uplinks.spec.ts`
5.x:  ...donno
perhaps its irrelevant in 5.x ❓ ❓ ❓ 

master: `packages/config/test/utils/index.ts`
5.x:  `test/unit/modules/utils/utils.spec.ts` ..maybe ❓ 
but it does not seem to correlate.
perhaps its irrelevant in 5.x ❓ ❓ ❓ 

master: `packages/config/test/utils/parse-configuration-file.ts`
5.x:  ...donno do not have a clue.

**Reply by osher** [reply]
I mean - help me with this mapping. 
if something is irrelevant - LMK.

**Comment by osher** [comment, marked as answer]
oh well. 
https://github.com/verdaccio/verdaccio/pull/2993
lets just continue there
